crates/server/openapi.yaml

```diff
@@ -216,54 +210,35 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-  /package/{logId}/record/{recordId}/content/{digest}:
-    post:
-      summary: Post content for a package record.
-      operationId: postPackageRecordContent
+  /content/{digest}:
+    get:
+      summary: Get content sources
+      operationId: getContentSources
       security: []
       tags:
-        - package
+        - content
       description: |
-        Posts package record content directly to the registry.
-
-        The package record must be in the `sourcing` state and the digest of the uploaded
-        content must match the digest parameter.
+        Gets a content sources for the given digest from the registry.
       parameters:
-        - name: logId
-          in: path
-          description: The package log identifier.
-          required: true
-          schema:
-            "$ref": "#/components/schemas/AnyHash"
-        - name: recordId
-          in: path
-          description: The record identifier.
-          required: true
-          schema:
-            "$ref": "#/components/schemas/AnyHash"
         - name: digest
           in: path
-          description: The digest of the content being uploaded.
+          description: The content digest.
           required: true
           schema:
             "$ref": "#/components/schemas/AnyHash"
-      requestBody:
-        content:
-          application/octet-stream:
-            schema:
-              type: string
-              format: binary
-              maxLength: 1073741824 # 1 GiB limit
+        - name: X-Proxy-Registry
+          in: header
+          description: The proxy registry URI Authority context to evaluate the request instead of the current registry.
+          required: false
+          schema:
+            type: string
```

Comment:
Out of scope for this PR?

Reply:
Oops, yeah that wasn't supposed to be in there. Pulling things out of another PR. I'll fix.